week-1/mandatory/2-dev-tools/task.md

```diff
@@ -6,12 +6,12 @@ In Google Chrome, open dev tools network tab and enter this URL into the browser
 
 Answer the following questions:
 
-- What's the status code the server sent back?
-- What HTTP method did the browser use to make the request?
-- What is the request path?
-- What is the first line of the response body?
-- What is the value of the response header called "Content-Type" ?
-- What is the value of the request header called "User-Agent"
+- What's the status code the server sent back? for episodes :304 Not Modified, for options.png:200 OK (from disk cache)
+- What HTTP method did the browser use to make the request? Get
+- What is the request path? http://api.tvmaze.com/shows/82/episodes
+- What is the first line of the response body?id: 4952,
+- What is the value of the response header called "Content-Type" ?image/png
```

Comment:
Make sure you're looking at the response for episodes🙂 The `Content-Type` is `application/json`.